src/taskClass.dart

```diff
@@ -0,0 +1,18 @@
+class TaskListApp {
+  // TODO: Implement Task list app
+  int id = 0;
+  String title = "";
+  String description = "";
+  bool isCompleted = false;
+  static int count = 0;
+  static Set<TaskListApp> tasks = {};
+
+  TaskListApp(String title, String description) {
+    this.id = count;
+    this.title = title;
+    this.description = description;
+    this.isCompleted = false;
+    count++;
+    tasks.add(this);
+  }
```

Comment:
No separation between task model and list. Should probably make a new class for the model and then use that here
```dart
class TaskModel{
    int;
    String title;
    String description;
    bool isCompleted;

    TaskModel({
        this.id = 0,
        this.title = "",
        this.description = "",
        this.isComplete = false,
    })
}

class TaskListApp
{
    List<TaskModel> tasks;

    void addTask(String title, String description)
    {
        TaskModel newTask = TaskModel(id: tasks.length, title: title, description: description);
        tasks.add(newTask);
    }
}
```